### PR TITLE
Removed empty package.json and package-lock.json from root

### PR DESCRIPTION
**Pull request by KeManen**

### CODE REVIEW

File: package.json
Comment (EddieTheCubeHead) [inline package.json:1]: I am not big into UI but to my understanding package.json is required to be in source control (like requirements.txt) while package-lock is not?

Comment (EddieTheCubeHead) [reply package.json:1]: @tuomasmustakallio @OttoAhlfors can you confirm or deny?

**Review by EddieTheCubeHead — CHANGES_REQUESTED** [review]

File: package.json
Comment (EddieTheCubeHead) [reply package.json:1]: But it might be only for packages? In which case it is indeed redundant for us

Comment (KeManen) [reply package.json:1]: Note the package.json and package-lock.json are from the root folder not the client/ folder.

Comment (EddieTheCubeHead) [reply package.json:1]: Ah

**Review by EddieTheCubeHead — APPROVED** [review]

**Review by tuomasmustakallio — APPROVED** [review]

**Review by BadhonRahman01 — APPROVED** [review]
!

**Pull request merged**